test/manipulation.test.js

```diff
@@ -1622,6 +1622,20 @@ describe('manipulation', function() {
   });
 
   describe('destroy', function() {
+    beforeEach(function resetFixtures(done) {
+      Person.destroyAll(function() {
+        Person.create([
+          {name: 'John'},
+          {name: 'Jane'},
+          {name: 'Mary'},
+        ], done);
+      });
+    });
+
+    after(function deleteFixtures(done) {
+      Person.destroyAll(done);
```

Comment:
Cleaning the database after the tests is an antipattern, see https://loopback.io/doc/en/lb4/Testing-your-application.html#clean-the-database-before-each-test

> Start with a clean database before each test. This may seem counter-intuitive: why not reset the database after the test has finished? When a test fails and the database is cleaned after the test has finished, then it’s difficult to observe what was stored in the database and why the test failed. When the database is cleaned in the beginning, then any failing test will leave the database in the state that caused the test to fail.